examples/function_memory_game/src/components/chessboard_card.rs

```diff
@@ -42,7 +42,7 @@ pub fn ChessboardCard(props: &Props) -> Html {
 
     html! {
       <div class="chess-board-card-container">
-          <div class={classes!("card", flipped.then(|| "flipped"))} {onclick}>
+          <div class={classes!("card", flipped.then_some("flipped"))} {onclick}>
```

Comment:
then_some is stable since 1.62, our MSRV is 1.60. However, I don’t think examples are run with MSRV clippy, so this should be fine.